packages/E2ETest/jest.setup.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ * @ts-check
+ */
+
+const fs = require('fs');
+const path = require('path');
+const sanitizeFilename = require('sanitize-filename');
+
+const screenshotDir = './errorShots';
+fs.mkdirSync(screenshotDir, {recursive: true});
+
+// Register to screenshot on each test failure
+global.jasmine.getEnv().addReporter({
```

Comment:
is 'jasmine' availalbe under 'jest' ?

Reply:
jest has some compatibility shims for Jasmine matchers and reporters.

Looking closer though, Jest has a mechanism for custom reporter inclusion that should be nicer... Will move this to native jest.